paasta_tools/cli/cmds/validate.py

```diff
@@ -308,35 +309,61 @@ def validate_chronos(service_path):
             "identifier used for temporary jobs" % TMP_JOB_IDENTIFIER
         ))
         return False
+    check_msgs = []
     for cluster in list_clusters(service, soa_dir, instance_type):
-        services_in_cluster = get_services_for_cluster(cluster=cluster, instance_type='chronos', soa_dir=soa_dir)
+        services_in_cluster = get_services_for_cluster(cluster=cluster, instance_type=instance_type, soa_dir=soa_dir)
         valid_services = {f"{name}{chronos_spacer}{instance}" for name, instance in services_in_cluster}
         for instance in list_all_instances_for_service(
                 service=service, clusters=[cluster], instance_type=instance_type,
                 soa_dir=soa_dir,
         ):
             cjc = load_chronos_job_config(service, instance, cluster, False, soa_dir)
             parents = cjc.get_parents() or []
-            checks_passed, check_msgs = cjc.validate()
```

Comment:
Hmm. I've renamed this function to `validate_chronos_dependencies`, because that kind of validation can only happen at the service level, and can't really happen on a validate() method on a single job.